libyul/optimiser/ForLoopConditionIntoBody.cpp

```diff
@@ -58,7 +58,7 @@ void ForLoopConditionIntoBody::operator()(ForLoop& _forLoop)
 			Literal {
 				debugData,
 				LiteralKind::Boolean,
-				"true"_yulstring,
+				LiteralValue{true, "true"},
```

Comment:
```suggestion
				LiteralValue{true},
```

Reply:
Looks like some tests are failing in CI because `true` gets replaced with `1` now. But it can't be this change, can it? It's a boolean literal and those should be formatted as `true`/`false` without a hint.